terraform/api_gateway.tf.json.template.py

```diff
@@ -193,17 +193,55 @@ def for_domain(cls, domain):
                     'rule': [
                         {**rule, 'priority': i}
                         for i, rule in enumerate([
+                            *([] if config.waf_file_download_limit is None else [{
+                                'name': 'FileDownloadRateLimit',
+                                'statement': {
+                                    'rate_based_statement': {
+                                        'limit': config.waf_file_download_limit[0],
```

Comment:
I think we should extract a module-level variable for `config.waf_file_download_limit`.